tools/next_gen_conversion/solid2fastq.py

```diff
@@ -1,15 +1,10 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
We can't do that without adding a `<requirement />` for some version of Python 3 to `tools/next_gen_conversion/solid2fastq.xml` and bumping the tool version.

Pretty sure this tool is abandonware, so probably not worth the effort, I'd just tighten the `# type: ignore` here as well.